api/gateway.proto

```diff
@@ -148,7 +148,14 @@ message Gateway {
   // The entire field must be used in RPCs since sub-fields are validated wrt to each other. Direct selection/update of sub-fields only are not allowed.
   // Use the top level field mask `claim_authentication_code` even when updating single fields.
   GatewayClaimAuthenticationCode claim_authentication_code = 23;
-  // next: 24
+  // CUPS URI for LoRa Basics Station CUPS redirection.
+  // The CUPS Trust field will be automatically fetched from the cert chain presented by the target server.
+  string target_cups_uri = 24 [(gogoproto.customname) = "TargetCUPSURI", (validate.rules).string.pattern = "^(?:(?:[a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\\-]*[a-zA-Z0-9])\\.)*(?:[A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\\-]*[A-Za-z0-9])(?::[0-9]{1,5})?$|^$"];
```

Comment:
- Is there a specific reason for not allowing `https://` in front? Without a scheme, we can't really call this a URI.
- If not, can we just use a `(validate.rules).string.uri = true` here?

Reply:
`https` was implied but good to make that explicit.